srtcore/core.cpp

```diff
@@ -9770,9 +9770,8 @@ bool srt::CUDT::overrideSndSeqNo(int32_t seq)
     // the latter is ahead with the number of packets already scheduled, but
     // not yet sent.
 
-    HLOGC(gslog.Debug, log << CONID() << "overrideSndSeqNo: sched-seq=" << m_iSndNextSeqNo << " send-seq=" << m_iSndCurrSeqNo
-        << " (unchanged)"
```

Comment:
`m_iSndCurrSeqNo` is also set in the above `setInitialSndSeq()`, so rm `unchanged`

Reply:
"Unchanged" means that scheduling sequence and sending sequence are the same and the sending sequence was not forcefully set to the value of scheduling sequence. Not sure if this could be anyhow clarified.

Reply:
OK, added `unchanged` back